### PR TITLE
support issues #6645 

### PR DESCRIPTION
**Pull request by andotorg**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

It is suggested to change the historical version query criteria box to the search drop-down box #6645

## Brief changelog

support issues #6645 

## Verifying this change

Valid

### CODE REVIEW

File: config/src/main/java/com/alibaba/nacos/config/server/controller/HistoryController.java [hunk 1]
```diff
@@ -143,5 +147,19 @@ public ConfigHistoryInfo getPreviousConfigHistoryInfo(@RequestParam("dataId") St
...
     }
+
+    /**
+     * Query configs list by namespace.
+     * @param tenant config_info namespace
+     * @since 2.1.1
+     * @return list
+     */
+    @GetMapping(value = "/configs")
+    public List<ConfigInfoWrapper> getDataIds(@RequestParam("tenant") String tenant) {
```
Comment (KomachiSion) [inline config/src/main/java/com/alibaba/nacos/config/server/controller/HistoryController.java:158]: add secure annotation to add auth check.

File: config/src/main/java/com/alibaba/nacos/config/server/service/repository/extrnal/ExternalStoragePersistServiceImpl.java [hunk 2]
```diff
@@ -2786,5 +2786,22 @@ public int tenantInfoCountByTenantId(String tenantId) {
...
+    public List<ConfigInfoWrapper> queryConfigInfoByNamespace(String tenant) {
+        Assert.hasText(tenant, "tenant can not be null");
+        String tenantTmp = StringUtils.isBlank(tenant) ? StringUtils.EMPTY : tenant;
+        try {
+            return this.jt.query(
+                    "SELECT data_id,group_id,tenant_id,app_name,type FROM config_info_beta WHERE tenant_id=?",
+                    new Object[]{tenantTmp},
+                    CONFIG_INFO_WRAPPER_ROW_MAPPER);
+        } catch (EmptyResultDataAccessException e) { // Indicates that the data does not exist, returns null.
+            return null;
```
Comment (KomachiSion) [inline config/src/main/java/com/alibaba/nacos/config/server/service/repository/extrnal/ExternalStoragePersistServiceImpl.java:2800]: Return empty list will be better.

**Review by KomachiSion — CHANGES_REQUESTED** [review]
I have a question about this feature.

According to previous feedback, users prefer to use the fuzzy search function, and this PR will cause the fuzzy search function to be abandoned. So I tend to keep input not select.

**Comment by andotorg** [comment]
How can do it now？

File: config/src/main/java/com/alibaba/nacos/config/server/service/repository/extrnal/ExternalStoragePersistServiceImpl.java (hunk 2 above)
Comment (andotorg) [reply config/src/main/java/com/alibaba/nacos/config/server/service/repository/extrnal/ExternalStoragePersistServiceImpl.java:2800]: why?

Comment (andotorg) [reply config/src/main/java/com/alibaba/nacos/config/server/service/repository/extrnal/ExternalStoragePersistServiceImpl.java:2800]: frontend not do return empty list and search keyword

File: config/src/main/java/com/alibaba/nacos/config/server/controller/HistoryController.java (hunk 1 above)
Comment (andotorg) [reply config/src/main/java/com/alibaba/nacos/config/server/controller/HistoryController.java:158]: > I have a question about this feature.
> 
> According to previous feedback, users prefer to use the fuzzy search function, and this PR will cause the fuzzy search function to be abandoned. So I tend to keep input not select.
this select is  have fuzzy search function

Comment (andotorg) [reply config/src/main/java/com/alibaba/nacos/config/server/controller/HistoryController.java:158]: According to different inputs, the corresponding matching data can be fuzzy searched

File: config/src/main/java/com/alibaba/nacos/config/server/service/repository/extrnal/ExternalStoragePersistServiceImpl.java (hunk 2 above)
Comment (andotorg) [reply config/src/main/java/com/alibaba/nacos/config/server/service/repository/extrnal/ExternalStoragePersistServiceImpl.java:2800]: i know。

Comment (andotorg) [reply config/src/main/java/com/alibaba/nacos/config/server/service/repository/extrnal/ExternalStoragePersistServiceImpl.java:2800]: select add showSearch property

File: config/src/main/java/com/alibaba/nacos/config/server/controller/HistoryController.java (hunk 1 above)
Comment (KomachiSion) [reply config/src/main/java/com/alibaba/nacos/config/server/controller/HistoryController.java:158]: Please Add the Secure annotation to add auth check.

**Comment by KomachiSion** [comment]
Author has no response and no change according the comment and review. So Close PR, if author or others want to continue contribute, please submit again.

**Review by HaTin79 — COMMENTED** [review]
Rất tốt, cám ơn

**Pull request closed without merging**